src/chatlistmodel.cpp

```diff
@@ -35,15 +36,21 @@ namespace {
     const QString CONTENT("content");
     const QString LAST_MESSAGE("last_message");
     const QString SENDER_USER_ID("sender_user_id");
+    const QString BASIC_GROUP_ID("basic_group_id");
+    const QString SUPERGROUP_ID("supergroup_id");
     const QString UNREAD_COUNT("unread_count");
     const QString NOTIFICATION_SETTINGS("notification_settings");
     const QString LAST_READ_INBOX_MESSAGE_ID("last_read_inbox_message_id");
     const QString LAST_READ_OUTBOX_MESSAGE_ID("last_read_outbox_message_id");
     const QString TYPE_MAP("type");
     const QString IS_CHANNEL("is_channel");
 
-    const QString TYPE("@type");
+    const QString _TYPE("@type");
```

Comment:
Well, there was a TYPE_MAP constant in there, but I can adjust that later as well... ;)